@navikt/core/react/src/form/combobox/Input/Input.tsx

```diff
@@ -132,17 +141,24 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
           if (activeDecendantId || value) {
             e.preventDefault();
           }
+        } else if (["ArrowLeft", "ArrowRight"].includes(e.key)) {
+          if (value !== "" && value !== searchTerm) {
+            onChange(e);
```

Comment:
Maybe add a comment about why we want to trigger onChange on ArrowLeft, ArrowRight and ArrowUp.